docs/topics/pages.md

```diff
@@ -0,0 +1,504 @@
+
+This should be a list of `SearchField` and `FilterField` objects. `SearchField` adds a field for full-text search. `FilterField` adds a field for filtering the results. A field can be indexed with both `SearchField` and `FilterField` at the same time (but only one instance of each).
+
+In the above example, we've indexed `body` for full-text search and `date` for filtering.
+
+The arguments that these field types accept are documented in [indexing extra fields](wagtailsearch_indexing_fields).
+
+### Editor panels
+
+There are a few attributes for defining how the page's fields will be arranged in the page editor interface:
+
+-   `content_panels` - For content, such as main body text
+-   `promote_panels` - For metadata, such as tags, thumbnail image and SEO title
+-   `settings_panels` - For settings, such as publish date
+
+Each of these attributes is set to a list of `EditHandler` objects, which defines which fields appear on which tabs and how they are structured on each tab.
+
+Here's a summary of the `EditHandler` classes that Wagtail provides out of the box. See [Panel types](/reference/pages/panels) for full descriptions.
+
+**Basic**
+
+These allow editing of model fields. The `FieldPanel` class will choose the correct widget based on the type of the field, though `StreamField` fields need to use a specialised panel class.
+
+-   [`FieldPanel`](/reference/pages/panels.html#wagtail.admin.edit_handlers.FieldPanel)
+-   [`StreamFieldPanel`](/reference/pages/panels.html#wagtail.admin.edit_handlers.StreamFieldPanel)
+
+**Structural**
+
+These are used for structuring fields in the interface.
+
+-   [`MultiFieldPanel`](/reference/pages/panels.html#wagtail.admin.edit_handlers.MultiFieldPanel) - For grouping similar fields together
+-   [`InlinePanel`](/reference/pages/panels.html#wagtail.admin.edit_handlers.InlinePanel) - For inlining child models
+-   [`FieldRowPanel`](/reference/pages/panels.html#wagtail.admin.edit_handlers.FieldRowPanel) - For organising multiple fields into a single row
+
+**Chooser**
+
+`ForeignKey` fields to certain models can use one of the below `ChooserPanel` classes. These add a nice modal chooser interface, and the image/document choosers also allow uploading new files without leaving the page editor.
+
+-   [`PageChooserPanel`](/reference/pages/panels.html#wagtail.admin.edit_handlers.PageChooserPanel)
+-   [`ImageChooserPanel`](/reference/pages/panels.html#wagtail.images.edit_handlers.ImageChooserPanel)
+-   [`DocumentChooserPanel`](/reference/pages/panels.html#wagtail.documents.edit_handlers.DocumentChooserPanel)
+-   [`SnippetChooserPanel`](/reference/pages/panels.html#wagtail.snippets.edit_handlers.SnippetChooserPanel)
+
+```eval_rst
+.. note::
+
+    In order to use one of these choosers, the model being linked to must either be a page, image, document or snippet.
+
+    Linking to any other model type is currently unsupported, you will need to use ``FieldPanel`` which will create a dropdown box.
```

Comment:
I actually don't think I changed that. I think that was changed before my conversion. I had a previous PR that got stuck for a long time because I made editorial changes like that. So I stopped doing that and focused only on the format conversion.

As for linking to "third-party" packages, I am not sure if that is done in the documentation. I see it is a Wagtail namespace package, but there is probably a reason it is not yet part of the core. 

Also, as described above, it's probably better to stay away from the editorial changes as part of the format conversion. Probably a good point for a follow up PR though.